example_logger_test.go

```diff
@@ -0,0 +1,146 @@
+package eris_test
```

Comment:
This example seems to big for one read. One suggestion is to break it down and explain part by part in docs or shorten it